components/SearchInput.vue

```diff
@@ -312,25 +312,24 @@ export default {
       function maybeParam(key, value) {
         return value && { [key]: `%${value}%` }
       }
-      this.$axios
-        .get('/api/students/', {
-          params: {
-            course: this.courseTag,
-            page: this.page,
-            sort: this.sortField,
-            order: this.sortOrder === 'asc' ? 'ASC' : 'DESC',
-            isActive: this.isActive !== 'AllStudents' ? this.isActive : null,
-            mailingList: this.mailingList !== '' ? this.mailingList : null,
-            isConcluding: !isNaN(this.isConcluding) ? this.isConcluding : null,
-            isForming: !isNaN(this.isForming) ? this.isForming : null,
-            isFit: this.isGraduating && !isNaN(this.isFit) ? this.isFit : null,
-            isGraduating: !isNaN(this.isGraduating) ? this.isGraduating : null,
-            ...maybeParam('name', this.searchName),
-            ...maybeParam('registrationNumber', this.searchRegistration),
-            ...maybeParam('email', this.searchEmail),
-            ...(this.blankCrgFilter && { noCrg: true })
-          }
-        })
+      const params = {
+        course: this.courseTag,
+        page: this.page,
+        sort: this.sortField,
+        order: this.sortOrder === 'asc' ? 'ASC' : 'DESC',
```

Comment:
Não precisa forçar maiúscula, o backend aceita minúscula

Reply:
resolvido

Reply:
Deu push?

Reply:
Dei agora.

Reply:
Mudou o problema